simpleimage/make_rootfs.sh

```diff
@@ -390,6 +423,78 @@ EOF
 		rm -f "$DEST"/etc/ssh/ssh_host_*
 		do_chroot ln -s /run/resolvconf/resolv.conf /etc/resolv.conf
 		;;
+	opensuse)
+		cat > "$DEST/etc/resolv.conf" << "EOF"
+nameserver 8.8.8.8
+nameserver 8.8.4.4
+EOF
+
+		cat > "$DEST/second-phase" <<EOF
+#!/bin/sh
+zypper remove -y kernel-default kernel-firmware
+zypper update
+zypper install -y dosfstools rfkill newt xterm patterns-openSUSE-yast2_basis
+zypper ar -f http://files.pine64.org/opensuse/repository "Pine64 support packages for openSUSE"
+zypper --gpg-auto-import-keys refresh
+passwd << END
+pine64root
+pine64root
+END
+useradd -m pine64
+passwd pine64 << END
+pine64linux
+pine64linux
+END
+groupadd sudo
+groupadd adm
+groupadd plugdev
+usermod -a -G sudo,adm,input,video,plugdev pine64
+zypper clean -a
+EOF
+		chmod +x "$DEST/second-phase"
+		do_chroot /second-phase
+		cat > "$DEST/etc/hostname" <<EOF
+pine64
+EOF
+		cat > "$DEST/etc/hosts" <<EOF
+127.0.0.1 localhost
+127.0.1.1 pine64
+# The following lines are desirable for IPv6 capable hosts
+::1     localhost ip6-localhost ip6-loopback
+fe00::0 ip6-localnet
+ff00::0 ip6-mcastprefix
+ff02::1 ip6-allnodes
+ff02::2 ip6-allrouters
+EOF
+		# Install platform scripts
+		mkdir -p "$DEST/usr/local/sbin"
+		cp -av ./platform-scripts/* "$DEST/usr/local/sbin"
+		wget http://files.pine64.org/opensuse/tools/pine64-config/opensuse/pine64-config.sh -O $DEST/usr/local/sbin/pine64-config.sh		
```

Comment:
What changes to the platform-scripts? Can you add seperate pull requests for those. I want to avoid a extra custom repository.